tests/integration/security/rbac/v2/group/group_test.go

```diff
@@ -174,11 +169,12 @@ func TestGroupV2RBAC(t *testing.T) {
 			time.Sleep(60 * time.Second)
 
 			for _, tc := range cases {
-				testName := fmt.Sprintf("%s->%s:%s/%s",
+				testName := fmt.Sprintf("%s->%s:%s%s[%v]",
 					tc.Request.From.Config().Service,
 					tc.Request.Options.Target.Config().Service,
 					tc.Request.Options.PortName,
-					tc.Request.Options.Path)
+					tc.Request.Options.Path,
```

Comment:
you might be better off manually specifying the test names to be something more abstract.  Using the URLs like this may become hard to understand going forward.  Instead, using names like `<something>With<something>Should<Succeed/Fail>`.  You can specify the test name as part of the case struct.

Reply:
I included the `tc.ExpectAllowed` which tells us the expect response (either allow or deny). The path and other information give a quick hint about what is being tested, it allows us to debug the failed test simply using `curl`.

Reply:
I also think this is good for debugging tests. Maybe either `false` or `true`, we can change it to `deny` or `allow`, but it's a nice to have, we can add that later.